### PR TITLE
Jorge/langtonant with Lint and Lint:fix and creation of jlangtonsant.test.js file

### PR DESCRIPTION
**Pull request by jarmcode**
In this PR there is my langtonsant code with lint and lint:fix and also the jlangtonsant.test.js file

### CODE REVIEW

File: katas/langtonant/jorge/jlangtonsant.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+let grid;
+let x;
+let y;
+let antDir;
+
+//Define where the ant turns
+let ANTUP = 0;
+let ANTRIGHT = 1;
+let ANTDOWN = 2;
+let ANTLEFT = 3;
```
Comment (J0c4) [inline katas/langtonant/jorge/jlangtonsant.js:10]: These variables are ok, however, if used as constants, please define them with `const` instead

File: katas/langtonant/richard/main.js [hunk 2]
```diff
@@ -1,4 +0,0 @@
-
```
Comment (J0c4) [inline katas/langtonant/richard/main.js:1]: I think this file should not be deleted

File: katas/langtonant/richard/module1.js [hunk 3]
```diff
@@ -1,133 +0,0 @@
-
```
Comment (J0c4) [inline katas/langtonant/richard/module1.js:1]: Richard did a refactor, but still, maybe this should not be removed to prevent merge conflicts

**Review by J0c4 — CHANGES_REQUESTED** [review]
Implementation looks good, it seems to be working as expected. There are more tests that should be added for antWalk, moveForward, turnLeft, turnRight

**Comment by J0c4** [comment]
Also, please solve merge conflicts

**Review by J0c4 — CHANGES_REQUESTED** [review]
Looks good!

Please rollback changes to `package-lock.json` file, this was probably modified when `npm install` was executed. We can ignore those changes for now

**Comment by J0c4** [comment]
> Looks good!
> 
> Please rollback changes to `package-lock.json` file, this was probably modified when `npm install` was executed. We can ignore those changes for now

Hi @jarmcode , still need this update to merge

**Comment by J0c4** [comment]
Hi @jarmcode , any updates on this one? issues need to be fixed because it might break the build. Since its been a while for this PR, we will need to set tomorrow as a deadline so we can close it

**Comment by jarmcode** [comment]
hi, Jose!

Conflicts have been solved!

**Comment by J0c4** [comment]
Hi @jarmcode , unfortunately, looks like coverage/ folder has been added (this files are expected not to be added to the repo), so this would prevent the approval of this PR, along with the package-lock.json file that was not reverted.

As mentioned, lets close this one for now, but we can follow up in this topic in case you have questions.

**Pull request closed without merging**